cmd/downloads/config/downloads_config_test.go

```
@@ -0,0 +1,269 @@
+		basename := filepath.Base(spec.Path)
+		base := filepath.Join(cfg.TempDir, spec.Arch, spec.OperatingSystem, basename)
+
+		tarPath := base + ".tar"
+		if _, err := os.Stat(tarPath); err != nil {
+			t.Errorf("tar archive not found at %s: %v", tarPath, err)
+			continue
+		}
+		f, _ := os.Open(tarPath)
+		tr := tar.NewReader(f)
+		hdr, err := tr.Next()
+		f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)